src/char/inter.c

```diff
@@ -814,19 +814,27 @@ static int inter_config_read(const char* cfgName)
 	return 0;
 }
 
-// Save interlog into sql
+/**
+* Save interlog values to SQL
+* @param fmt: Pointer containing the type of activity to be stored
+*  Supplied arguments: activity, origin, target, guild pointer, castle_id
+* @return 0
+*/
 int inter_log(char* fmt, ...)
```

Comment:
I'm thinking that if we always pass the same amount of arguments over to the log function, why are we using va_arg? It would be best to just define all the parameters out.